surreal/client.py

```diff
@@ -0,0 +1,75 @@
+"""
+Copyright © SurrealDB Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from typing import Any
+
+from surreal.http import HTTPClient
+
+
+class Surreal:
+    """
+    This class wraps around the HTTP Client adding a few 
```

Comment:
```suggestion
    """This class wraps around the HTTP Client adding a few 
```
Extra blank lines in docstrings like this are unnecessary.